### ISSUE TITLE
Is this project still under progress?

### ISSUE THREAD

**Issue by haroldrandom** [graarh/golang-socketio#40]
Hi, there!

Is this project still under progress? Any updates?

**Comment by henvic** [comment]
I have no idea if @graarh has plans to update this.

I see I and others ended up rewriting portions of this to our own needs.
For example, @mtfelian did a major refactor on all parts (https://github.com/graarh/golang-socketio/issues/34), and I myself have extracted the client and refactored it a lot (http://github.com/wedeploy/gosocketio).

I don't have an interest in maintaining a fork of a Go library for socket.io (besides the necessary for my needs), but maybe it's about time someone merges our improvements together.

**Comment by haroldrandom** [comment]
@henvic Thank you for you efforts!

1. Is your repo already on service in the production environment? 

2. What version of socket.io server have you tested?

**Comment by mtfelian** [comment]
@haroldrandom  You can also try my fork
PRs and upgrades appreciated

**Comment by henvic** [comment]
Hi @haroldrandom,

If you go to https://asciinema.org/a/192043 between 1:57 and 2:15 you can see the execution of some shell commands. I'm actually using this modified socketio implementation behind the scenes for that.

There aren't too many users for this command, but I'd say it's pretty stable. That said, I must say there are a couple bugs which I believe are not related to golang-socketio.

**Comment by haroldrandom** [comment]
@mtfelian @henvic Thank you guys and your efforts.

I am trying to implement a socket.io client using the [gorilla websocket package](https://github.com/gorilla/websocket), and some problems comes to me. 
At first I consider it's about the websocket package I used,  but then I realized it is not. 

I just can't pass the user authorizing system, that's why I can't get data as the browser did.
Now everything is working fine with gorilla websocket package.

Thank you for yours suggestions and efforts again~
